worker/server_state.go

```diff
@@ -30,6 +30,10 @@ import (
 	"github.com/golang/glog"
 )
 
+const (
+	BadgerDefaults = "goroutines=8; compression=snappy; cache_mb=64; cache_percentage=70,30"
```

Comment:
Some of these defaults aren't used by Alpha, which is kind of confusing...

We use [`goroutines`](https://github.com/dgraph-io/dgraph/pull/7387/files#diff-a72445362e52218163e2d2f572b946ff3aea61f8675aa235a5dab3227ac6168eR113) and [`compression`](https://github.com/dgraph-io/dgraph/pull/7387/files#diff-a72445362e52218163e2d2f572b946ff3aea61f8675aa235a5dab3227ac6168eR114) in Alpha.

We use [`compression`](https://github.com/dgraph-io/dgraph/pull/7387/files#diff-76237e26c3354ceb21e16c8c34877ddb485e468d4eb711ba35477f35f9018e96R120), [`cache_mb`](https://github.com/dgraph-io/dgraph/pull/7387/files#diff-76237e26c3354ceb21e16c8c34877ddb485e468d4eb711ba35477f35f9018e96R123), and [`cache_percentage`](https://github.com/dgraph-io/dgraph/pull/7387/files#diff-76237e26c3354ceb21e16c8c34877ddb485e468d4eb711ba35477f35f9018e96R124) in Bulk.